readthedocs/urls.py

```diff
@@ -21,8 +21,12 @@
 from readthedocs.search.api import PageSearchAPIView
 
 
+from readthedocs.docsitalia.views.core_views import server_error_401
+
+
 admin.autodiscover()
 
+handler401 = server_error_401
```

Comment:
should we add `handler401` in `rtd_urls`?

Reply:
Done